ui/scss/component/_ads.scss

```diff
@@ -33,10 +33,17 @@
     aspect-ratio: 16 / 9;
     $minWidth: calc(var(--file-list-thumbnail-width) * 0.8);
     min-width: $minWidth;
+    background-color: #283263;
+    background-image: url('https://odysee.com/public/img/astronaut_n_friends.png');
+    background-size: 100%;
+    border-radius: var(--border-radius);
+    box-shadow: 0px 0px 0px 1px rgba(var(--color-primary-dynamic), 0.1) inset;
```

Comment:
webstorm says it''s redundant
(which by the way, we should probably add a css linter in the near future?)
![image](https://user-images.githubusercontent.com/64950861/168557478-c405b39c-c3ba-4ad4-8845-24c1dfa18f1f.png)

Reply:
It is redundant, but I usually only remove the units when all the values are 0. In this case there's still 1px for the bottom left. I can remove it tho...

Reply:
No worries, was just passing along info.
We follow https://github.com/airbnb/css style guide per https://github.com/OdyseeTeam/odysee-frontend/blob/master/CONTRIBUTING.md, and I always thought the rule was part of it, but seems like not.